inference-engine/src/transformations/include/transformations/common_optimizations/convert_nms_gather_path_to_unsigned.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <transformations_visibility.hpp>
+#include <ngraph/pass/pass.hpp>
+
+namespace ngraph {
+namespace pass {
+
+class TRANSFORMATIONS_API ConvertNmsGatherPathToUnsigned;
+
+}  // namespace pass
+}  // namespace ngraph
+
+/**
+ * Converts Gather indices to unsigned if indices are from NMS selected indices output.
```

Comment:
Please use doxygen labels. You can find them in any transformation.

Reply:
done